benchmarks/Getting_Started_Benchmarking.md

```diff
@@ -19,9 +19,9 @@ python3 -m benchmarks.benchmark_runner xpk --project $PROJECT --zone $ZONE --clu
 
 ```shell
 # Pathways with XPK
-export RUNNER=us-docker.pkg.dev/cloud-tpu-v2-images-dev/pathways/maxtext_jax_stable
-export PROXY_IMAGE=us-docker.pkg.dev/cloud-tpu-v2-images-dev/pathways/proxy_server:latest
-export SERVER_IMAGE=us-docker.pkg.dev/cloud-tpu-v2-images-dev/pathways/server:latest
+export RUNNER=us-docker.pkg.dev/path/to/maxtext_runner
```

Comment:
Is the `/path/to` intentional here and in the other files? What does this mean?

Reply:
Yes this is intentional, we want the user to put in their own maxtext image.